content/partials/types/_error_info.textile

```diff
@@ -17,3 +17,133 @@ h4.
 
 blang[flutter].
   - requestId :=  Request ID with which the error can be identified<br>__Type: @String@__
+
+h4. Error nesting
+
+ErrorInfo objects can contain nested errors through the @cause@ property, allowing you to trace the root cause of failures. When an operation fails due to underlying system errors, the main @ErrorInfo@ provides the high-level failure reason while the nested @cause@ contains more specific details about what went wrong.
+
+A common nesting scenario is "80019: Auth server rejecting request":/docs/platform/errors/codes#80019 where the main error indicates token renewal failed, while the nested @cause@ contains the specific HTTP error from the auth server.
+
+The following example demonstrates how to handle nested errors:
+
+blang[javascript].
```

Comment:
I think you still need to set the language on the codeblock for it to render highlighting correctly. I'm not sure if you need the blangs in addition to that though - might be worth trying without them.